_tests/integration/step_info_test.go

```diff
@@ -195,6 +195,9 @@ type_tags:
 is_requires_admin_user: true
 is_always_run: false
 is_skippable: false
+meta:
+  bitrise.io.addons.required: [{addon_id:"addons-testing"}]
```

Comment:
This is not a valid YML, here you defined the `bitrise.io.addons` with JSON values, which is fine, but `[{addon_id:"addons-testing"}]` is not valid JSON, that's why the key is missing spotted by Tomi: https://github.com/bitrise-io/stepman/pull/268/files#r270055538

`[{addon_id:"addons-testing"}]` -> ``[{"addon_id":"addons-testing"}]``

Other than that I would recommend specifying a bit more complex meta to tests if further options will be kept and parsed properly:

```
meta:
  bitrise.io.addons.required: 
    - addon_id: "addons-testing"
      addon_params: "--token TOKEN"
      addon_options: 
        required: true
        title: "Testing Addon"
    - addon_id: "addons-ship"
      addon_params: "--token TOKEN"
      addon_options: 
        required: true
        title: "Ship Addon"
  bitrise.io.addons.optional: [{"addon_id":"addons-testing"}] # json value
```